test/services/twilio-help/twilio-help.test.js

```diff
@@ -50,15 +50,13 @@ const testHelp = test
 describe('should log help', () => {
   testHelp.commandHelp([], true).it('Should display required flags', (ctx) => {
     expect(ctx.commandHelp).to.contain('api:testHelp');
-    expect(ctx.commandHelp).to.contain('OPTIONS');
-    expect(ctx.commandHelp).to.contain('REQUIRED FLAGS\n  -f, --foo=foo');
+    expect(ctx.commandHelp).to.contain('REQUIRED FLAGS\n  -f, --foo=');
     expect(ctx.commandHelp).to.contain('OPTIONAL FLAGS');
     expect(ctx.commandHelp).to.contain('USAGE');
   });
 
   testHelp.commandHelp([]).it('Should not display required flags if not configured', (ctx) => {
     expect(ctx.commandHelp).to.contain('api:testHelp');
-    expect(ctx.commandHelp).to.contain('OPTIONS');
```

Comment:
[self] There's change in the new format. It no longer says "OPTIONS" before listing out flags. 
It can be added based on feedback. 

<img width="834" alt="image" src="https://user-images.githubusercontent.com/87537688/187089932-5e12e604-66d6-4c28-b14b-835873fa2101.png">

Reply:
I agree its not needed since we are already dividing options based on required/ optional flags.